flexget/api/core/server.py

```diff
@@ -8,6 +8,8 @@
 import traceback
 from pathlib import Path
 from time import sleep
+from typing import Dict, IO, Generator
+from typing import Optional as OptionalType
```

Comment:
why alias it?

Reply:
`Optional` is imported from pyparsing on [L.24](https://github.com/Flexget/Flexget/blob/d18633038cb9d75661305b4146c0aaee6d92edde/flexget/api/core/server.py#L24)